README.md

```diff
@@ -64,8 +64,8 @@ Packages in the Coq ecosystem are managed by the [opam package manager](https://
 
 We need a custom build and **cannot** use opam for this step.
 
-1. `wget -O - https://github.com/PrincetonUniversity/VST/archive/v2.11.tar.gz | tar -xvzf -`
-1. `cd VST-2.11`
+1. `wget -O - https://github.com/PrincetonUniversity/VST/archive/v2.12.tar.gz | tar -xvzf -`
+1. `cd VST-2.12`
 1. `make -j$(nproc) default_target sha`
```

Comment:
```
$ make -j$(nproc) default_target sha
Makefile:131: *** COMPCERT VERSION MISMATCH: COMPCERT_VERSION=version=3.12 but /home/user/.opam/default/lib/coq/user-contrib/compcert/VERSION=version=3.11.  Stop.
```

I had ignore this conflict, like in nix. The compcert version seems to be fixed to the Coq version, so I don't see another workaround.

```
export IGNORECOMPCERTVERSION=true
```